modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -80,23 +80,22 @@ export class MessengerService {
           .error('You need to configure an Access Token to enable it. Messenger Channel is disabled for this bot.')
       }
 
-      const { data } = await this.http.get('/', { params: { access_token: config.accessToken } })
-
-      if (!data?.id) {
+      try {
+        const { data } = await this.http.get('/', { params: { access_token: config.accessToken } })
+
+        const pageId = data.id
+        const client = new MessengerClient(botId, this.bp, this.http)
+        this.mountedBots.push({ botId, client, pageId })
+
+        await client.setupGreeting()
+        await client.setupGetStarted()
+        await client.setupPersistentMenu()
+      } catch (error) {
+        const errorMessage = _.get(error, 'response.data.error.message', 'are you sure your Access Token is valid?')
         return this.bp.logger
           .forBot(botId)
-          .error(
-            'Could not register bot, are you sure your Access Token is valid? Messenger Channel is disabled for this bot.'
-          )
+          .error('Could not register bot, ' + errorMessage + '. Messenger Channel is disabled for this bot.')
```

Comment:
'Could not register bot, ' + errorMessage + '. Messenger Channel is disabled for this bot.'

use template string please